CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpImportViewController.swift

```diff
@@ -0,0 +1,122 @@
+        view.backgroundColor = UIColor.from(giniColor: giniConfiguration.helpScreenBackgroundColor)
+        edgesForExtendedLayout = []
+    }
+    
+    private func configureTableView() {
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.separatorStyle = .none
+        tableView.backgroundColor = UIColor.clear
+        tableView.tableFooterView = UIView()
+        tableView.register(UINib(nibName: "HelpImportCell", bundle:giniCaptureBundle()), forCellReuseIdentifier: HelpImportCell.reuseIdentifier)
+        tableView.contentInsetAdjustmentBehavior = .never
+    }
+    
+    private func configureConstraints() {
+        view.addConstraints([
+            tableView.topAnchor.constraint(equalTo: view.topAnchor, constant: margin),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: margin),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -margin),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+    }
+    
+}
+
+extension HelpImportViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        let itemType = dataSource[indexPath.row]
+        switch itemType {
+        case .importToApp:
+            return 350
+        case .selectInvoice:
+            return 300
+        }
+    }
+}
+
+extension HelpImportViewController: UITableViewDataSource {
+    
+    private func configureCell(cell: HelpImportCell, indexPath: IndexPath) {
+        let itemType = dataSource[indexPath.row]
+        switch itemType {
+        case .selectInvoice:
+            cell.headerLabel.text = "1. Select an invoice"
+            cell.descriptionLabel.text = "To do so, please select a PDF invoice from within your email app, PDF viewer or other app on your smarthpone. To redirect the file to GiniVision, use the “Share” function, represented as a square with the arrow pointing up."
+            cell.importImageView.image = UIImageNamedPreferred(named: "helpImport1")
+        case .importToApp:
+            cell.headerLabel.text = "2. Import to app"
+            cell.descriptionLabel.text = "Please select your Banking App from the list to start the analysis and transfer process."
+            cell.importImageView.image = UIImageNamedPreferred(named: "helpImport2")
+        }
+        cell.backgroundColor = UIColor.clear
+        cell.headerLabel.backgroundColor = UIColor.clear
+        cell.descriptionLabel.backgroundColor = UIColor.clear
+        cell.contentView.backgroundColor = UIColor.clear
+    }
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        if let cell = tableView.dequeueReusableCell(withIdentifier: HelpImportCell.reuseIdentifier) as? HelpImportCell {
+            configureCell(cell: cell, indexPath: indexPath)
+            return cell
+        }
+        fatalError()
```

Comment:
@chriskryn would you mind adding a message for this fatal error 
![image](https://user-images.githubusercontent.com/3911816/182868746-4200b120-6c9b-4697-b8c2-026cdfcbec65.png)